.github/workflows/semantic-pr-check.yml

```diff
@@ -0,0 +1,17 @@
+name: "Semantic PR Check"
```

Comment:
Could you change the file name to `on-pull-request-check-pr-title`? The convention I'm trying to set up for workflow files is to have the trigger be a file name prefix.

Reply:
I like this convention and will try and keep it in mind.